test/e2e/constants/constants.go

```diff
@@ -28,6 +28,7 @@ var (
 	RunLocalOperator  = strings.ToUpper(constants.GetEnvWithDefault("RUN_LOCAL_OPERATOR", "true"))
 	RunSaOperator     = strings.ToUpper(constants.GetEnvWithDefault("RUN_SA_OPERATOR", "false"))
 	ImageName         = constants.GetEnvWithDefault("IMAGE", "infinispan/server:12.0")
+	NativeImageName   = constants.GetEnvWithDefault("NATIVE_IMAGE", "quay.io/infinispan/server-native:12.0")
```

Comment:
We should either use `Quay.io` or docker hub for both images. I don't mind which, but we should be consistent with `ImageName` and `NativeImageName`.

Reply:
Yes, sure, @ryanemerson . What is docker hub native image location?

Reply:
`infinispan/server-native:12.0`